### ISSUE TITLE
Alien queen sprite does not appear fully on fire

### ISSUE THREAD

**Issue by bgobandit** [tgstation/-tg-station#12605]
What happened: Alium queen got set on fire via walking through a huge fire patch (so no question about whether it was aimed right). The fire overlay only covered, like, its foot or something, but the alium was on fire.

Why this is bad: Really hard to tell whether an alien queen is on fire; obvious repercussions.

**Comment by newfren** [comment]
I'm pretty sure this is because of WJ's sprite changes to the alien queen.

**Comment by GunHog** [comment]
WJ and I will probably address this when I do the sprite swap he requested.

**Comment by WJohn** [comment]
Yeah it's something I'll have to get to for both the queen and praetorian in the future. Not a huge deal right now though, even if it looks ugly.

**Comment by Bawhoppen** [comment]
I think its a problem with larva too.

**Comment by Qbopper** [comment]
Via lava

![lava](http://i.imgur.com/bnyDv2w.png)

via flamethrower

![flame](http://i.imgur.com/UBazgUi.png)

it's weird because I distinctly remember seeing an alien queen lit up with proper fire sprites, but I guess I might be insane

**Comment by GunHog** [comment]
I cannot do anything about this, as there are no fire overlays for the royals.

**Comment by Qbopper** [comment]
Would you need to have a different sprite designed to specifically overlay onto the alien, then code to make it use the right one or something?

**Comment by GunHog** [comment]
Correct. Have a look at https://github.com/tgstation/tgstation/blob/master/icons/mob/alienqueen.dmi . It has overlays for the royals that fit in their sprite at appropriate times.

**Comment by Qbopper** [comment]
Ah, so this is just waiting for a spriter to take care of it then

**Comment by dragomagol** [comment]
Still an issue

**Comment by SuperNovaa41** [comment]
Still an issue

**Comment by dragomagol** [comment]
![image](https://user-images.githubusercontent.com/66640614/168199201-221d6e96-4391-4ab4-8dcf-2eee1e8becc4.png)

😔

**Comment by Qbopper** [comment]
I don't play the game and yet I track the passage of time by amount of emails I get over this issue

The year is 2017. Alien queens do not appear on fire correctly.

The year is 2022. Alien queens do not appear on fire correctly.

**Comment by Bawhoppen** [comment]
@Qbopper Hi Qbopper how are you doing. I feel a similar way about issues; I'm subscribed to so many over the years, it always reminds me of where I was when the issue was first created. Good stuff.

**Comment by itseasytosee** [comment]
Still an issue. Needs some custom sprites.

**Comment by SethLafuente** [comment]
![image](https://github.com/tgstation/tgstation/assets/84478872/4709efbc-6cc3-4326-af3a-bea2e1f46ea9)
still a bug as of 2023

**Comment by JohnFulpWillard** [comment]
bug as of 2024
![image](https://github.com/tgstation/tgstation/assets/53777086/1abef03c-19a9-4a8c-86d8-7d7a67007372)

**Comment by DATA-xPUNGED** [comment]
ain't no fucking way

**Comment by JaneAtreides** [comment]
this issue has been open for so long that i transitioned before it got closed

i feel like i'm at a funeral or something

**Comment by Time-Green** [comment]
I didnt know this was here. We can add the bug back in

**Comment by Bawhoppen** [comment]
This was 11 years ago? Oh crap.
